cmd/worker/main.go

```diff
@@ -38,8 +50,17 @@ func main() {
 	}
 }
 
-func newLocalTemporalClient() (client.Client, error) {
-	return client.Dial(client.Options{})
+func newTemporalClient(logger *zap.Logger) (client.Client, error) {
+	ns := os.Getenv(temporalCloudNamespaceEnvName)
+	if ns == "" {
+		return client.Dial(client.Options{})
+	}
+	return temporal.GetTemporalCloudNamespaceClient(&temporal.GetTemporalCloudNamespaceClientInput{
+		Namespace:       ns,
+		TLSCertFilePath: os.Getenv(temporalCloudTLSCertPathEnv),
```

Comment:
We need to check if TLS Cert/Key are provided in env. If not we assume it isnt cloud and connect without TLS.

So something like:
if os.Getenv(temporalCloudTLSCertPathEnv) != "" && os.Getenv(temporalCloudTLSKeyPathEnv) != "" {
...
} else {
...
}